sbws/lib/v3bwfile.py

```diff
@@ -32,22 +32,44 @@
 
 # Header KeyValues
 # =================
-# List of the extra KeyValues accepted by the class
-EXTRA_ARG_KEYVALUES = ['software', 'software_version', 'file_created',
-                       'earliest_bandwidth', 'generator_started',
-                       'scanner_country', 'destinations_countries']
+# KeyValues that need to be in a specific order in the Bandwidth File.
+HEADER_KEYS_V1_1_ORDERED = ['version']
+# KeyValues that are not initialized from arguments nor depend on the state
+# file nor the measurements.
+HEADER_KEYS_V1_1_SELF_INIT = [
```

Comment:
The difference between SELF_INIT and TO_INIT is unclear, please rename.
I suggest "ARG" and "EXTERN". (Arguments or something external.)

I also wonder if latest_bandwidth does depend on the state file?
If there are new bandwidths (or no bandwidths at all), then the latest bandwidth should change.